documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -6,10 +6,42 @@
 [id='assembly-upgrade-{context}']
 = Upgrading Strimzi
 
-Strimzi can be upgraded from a lower version to a higher version to take advantage of new features and enhancements, performance improvements, and security options.
+Strimzi can be upgraded from a given version to a later version to take advantage of new features and enhancements, performance improvements, and security options.
 
-As part of the upgrade, you upgrade Kafka to a higher supported version. 
-Each Kafka release introduces new features, improvements, and bug fixes.
+As part of the upgrade, you upgrade Kafka to a later supported version. 
+Each Kafka release introduces new features, improvements, and bug fixes to your Strimzi deployment.
+
+Strimzi can be xref:assembly-downgrade-{context}[downgraded] to the previous version if you encounter issues with a newer version.
+
+.Upgrade paths
+
+Two upgrade paths are possible:
+
+Incremental:: 
+Upgrading Strimzi from the current version to the next version.
++
+For example, upgrading from Strimzi 0.20.1 to 0.21.0.
+
+Multi-version::
+Upgrading Strimzi from the current version to any later version within a single upgrade (skipping one or more intermediate versions).
```

Comment:
Again, this is documentation for Strimzi 0.22. So this is about upgrading _from older versions to current version_ or possibly _from older versions to Strimzi {ProductVersion}_.

Reply:
Changed to: